File: Documentation/4.0/Samples/nodejs/client-api/session/howTo/getLastModified.js
import * as assert from "assert";
import { DocumentStore } from "ravendb";

const store = new DocumentStore();
const session = store.openSession();

{
    let instance;
    //region get_last_modified_1
    session.advanced.getLastModifiedFor(instance);
    //endregion
}

async function sample() {
    {
        //region get_last_modified_2
        const employee = await session.load("employees/1-A");
        const lastModified = session.advanced.getLastModifiedFor(employee);
        //endregion
    }
}

File: Documentation/5.4/Samples/java/src/test/java/net/ravendb/ClientApi/Session/HowTo/GetLastModified.java
package net.ravendb.ClientApi.Session.HowTo;

import net.ravendb.client.documents.DocumentStore;
import net.ravendb.client.documents.IDocumentStore;
import net.ravendb.client.documents.session.IDocumentSession;

import java.util.Date;

public class GetLastModified {

    private static class Employee {
        private String firstName;
        private String lastName;

        public String getFirstName() {
            return firstName;
        }

        public void setFirstName(String firstName) {
            this.firstName = firstName;
        }

        public String getLastName() {
            return lastName;
        }

        public void setLastName(String lastName) {
            this.lastName = lastName;
        }
    }


    private interface IFoo {
        //region get_last_modified_1
        <T> Date getLastModifiedFor(T instance)
        //endregion
        ;
    }

    public GetLastModified() {
        try (IDocumentStore store = new DocumentStore()) {
            try (IDocumentSession session = store.openSession()) {
                //region get_last_modified_2
                Employee employee = session.load(Employee.class, "employees/1-A");
                Date lastModified = session.advanced().getLastModifiedFor(employee);
                //endregion
            }
        }
    }

}
